planner/core/task.go

```diff
@@ -1077,8 +1083,6 @@ func (t *copTask) convertToRootTaskImpl(ctx sessionctx.Context) *rootTask {
 		newTask.p = sel
 		sel.cost = newTask.cost()
```

Comment:
Here we add a new Selection operator, guess we should update newTask's cost?
`newTask.addCost(t.count() * sessVars.CPUFactor)` just like `PhysicalSelection.attach2Task` did.
@winoros